python_modules/dagster/dagster/_core/instance/persist_run_for_production.py

```diff
@@ -0,0 +1,71 @@
+from typing import Mapping, Optional
+
+from dagster._core.definitions.utils import validate_tags
+from dagster._core.host_representation.repository_location import RepositoryLocation
+from dagster._core.host_representation.selector import PipelineSelector
+from dagster._core.instance import DagsterInstance
+from dagster._core.storage.pipeline_run import DagsterRun, DagsterRunStatus
+from dagster._core.utils import make_new_run_id
+
+
+def persist_run_for_production(
```

Comment:
The word "production" rubs me the wrong way a little it, because this will often be invoked within environments that users would not designate as "production".

Thoughts on just calling this "create_run" and renaming the other one to something like "create_run_inner"?

Reply:
Sandy I'm going to respond to alex's comment (similar feedback) and resolve this thread.